File: archive/partitioning-into-minimum-number-of-deci-binary-numbers.js
const minPartitions = function(n) {
  n = n.split('')
  n = n.map(el => Number(el))

  let ans = 0
  for (let i = 0; i < n.length; i++) {
    let r = Math.max(n[i] - ans, 0)
    if (r > 0) {ans+=r}
  }
  return ans
};

console.log(minPartitions("27346209830709182346"))

File: go-archive/partitioning-into-minimum-number-of-deci-binary-numbers.go
func minPartitions(n string) int {
    arr:=strings.Split(n, ""); ans:=0

    for i:=0; i < len(arr); i++ {
        j, err := strconv.Atoi(arr[i])
        if (err != nil) {
            fmt.Println("Not current type")
        }
        if (j > ans) {ans = j}
    }
    return ans
}